src/components/Stage.js

```diff
@@ -190,13 +215,24 @@ class Stage extends Component {
             className="ui placeholder"
             style={{
               marginLeft: "2.5rem",
-              marginRight: "1.5em",
+              marginRight: "2em",
               height: "1.2em",
             }}
           >
             <div className="long line" style={{ backgroundColor: "initial" }} />
           </div>
           <div style={{ clear: "both" }} />
+          <div
+            className="ui icon button octopus-theme publication disabled"
+            style={{
+              padding: "0.5rem",
+              position: "absolute",
+              top: "calc(13px - (1.07142857rem * 1.28571429 - 13px) / 2)",

```

Comment:
These look like overly-specific values. Can we round them up?

Reply:
Done